app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -21,10 +21,15 @@ define([
       // there might not be any relier if the resume token is being fetched
       // for an account unlock request caused by changing the password.
       var relierInfo = this.relier && this.relier.pickResumeTokenInfo();
+      var userInfo = this.user && this.user.pickResumeTokenInfo();
 
       // When account or user fields are needed,
       // they can be added as part of the resumeTokenInfo
-      var resumeTokenInfo = _.extend({}, relierInfo);
+      var resumeTokenInfo = _.extend(
+        {},
+        relierInfo,
+        userInfo
```

Comment:
As @rfk pointed out before, merging relierInfo and userInfo into the same hash is prone to collisions. We could namespace the data into two keys, `relier`, and `user`. We would have to do the same in the ResumeToken model. This would require an update anywhere the ResumeToken's info is accessed.